pipeline/backend/kubernetes/service.go

```diff
@@ -15,6 +15,7 @@
 package kubernetes
 
 import (
+	"fmt"
```

Comment:
```suggestion
	"fmt"

```

Reply:
https://github.com/woodpecker-ci/woodpecker/pull/2933#discussion_r1428841966

Reply:
It must have newline between import groups std, local and external

Reply:
It should look better now